django_elasticsearch_dsl/search.py

```diff
@@ -13,12 +14,20 @@ def _clone(self):
         s._model = self._model
         return s
 
-    def to_queryset(self, keep_order=True):
+    def filter_queryset(self, klass, keep_search_order=True):
         """
-        This method return a django queryset from the an elasticsearch result.
-        It cost a query to the sql db.
+        Filter an existing django queryset using the elasticsearch result.
+        It costs a query to the sql db.
+        klass may be a Model, Manager, or QuerySet object.
         """
+
+        qs = _get_queryset(klass)
```

Comment:
I am not comfortable to use a private method `_get_queryset` as it is not documented anywhere.

I think it is better to use `self._model._default_manager` for getting the default manager as the [documentation](https://docs.djangoproject.com/en/3.0/topics/db/managers/#default-managers) recommend it.

Reply:
Done.